app/src/main/java/com/kinandcarta/create/proxytoggle/feature/manager/view/ProxyManagerFragment.kt

```diff
@@ -0,0 +1,66 @@
+package com.kinandcarta.create.proxytoggle.feature.manager.view
+
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.fragment.app.Fragment
+import androidx.fragment.app.viewModels
+import androidx.lifecycle.Observer
+import com.kinandcarta.create.proxytoggle.databinding.FragmentProxyManagerBinding
+import com.kinandcarta.create.proxytoggle.feature.manager.viewmodel.ProxyManagerViewModel
+import dagger.hilt.android.AndroidEntryPoint
+
+@AndroidEntryPoint
```

Comment:
why do you need this? you don't seem to be injecting anything

Reply:
Oh this was just a temporary Fragment for testing stuff

https://github.com/theappbusiness/android-proxy-toggle/pull/7#discussion_r448906522